lib/client/api.go

```diff
@@ -70,6 +71,7 @@ import (
 	"github.com/gravitational/teleport/lib/utils/agentconn"
 	"github.com/gravitational/teleport/lib/utils/prompt"
 	"github.com/gravitational/teleport/lib/utils/proxy"
+	"github.com/gravitational/teleport/tool/common"
```

Comment:
I've gotta say, I'm pretty uncomfortable with `lib` calling out `tool`. Generally we have `tool/<tool_name>` containing all the code specific to that tool. Common code shared between tools is always under `lib` or `api`. I'm okay with adding `tool/common` for shared code that is close to the CLI (like `WriteJSON`/`WriteYAML` helpers,) but making `lib` depend on `tool` seems quite backwards to the rest of the dependency graph.

`tool/common` also depends on other packages under `lib`, so someone is bound to make an innocuous change in `lib` that creates a dependency cycle, and this could get tricky to untangle. IMO `GetPaginatedSessions` should really be in `lib/client` and that would solve this.

@r0mant what do you think? @EdwardDowling sorry to keep suggesting moving things around.

Reply:
Yeah this makes sense, will move it now.